<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="6" failed="0" total="6" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="59576" started-at="2019-10-10T03:05:21Z" finished-at="2019-10-10T03:06:21Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="59576" started-at="2019-10-10T03:05:21Z" finished-at="2019-10-10T03:06:21Z">
      <class name="com.amazon.qa.testcases.LoginPageTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="setup" is-config="true" duration-ms="9175" started-at="2019-10-10T08:35:21Z" finished-at="2019-10-10T08:35:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="titletest()[pri:1, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="titletest" duration-ms="98" started-at="2019-10-10T08:35:30Z" finished-at="2019-10-10T08:35:30Z">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[Title does not match expected:<false> but was:<true>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: Title does not match expected:<false> but was:<true>
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.failNotEquals(Assert.java:282)
	at junit.framework.Assert.assertEquals(Assert.java:64)
	at junit.framework.Assert.assertEquals(Assert.java:149)
	at com.amazon.qa.testcases.LoginPageTest.titletest(LoginPageTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titletest -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="close" is-config="true" duration-ms="928" started-at="2019-10-10T08:35:30Z" finished-at="2019-10-10T08:35:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="setup" is-config="true" duration-ms="5899" started-at="2019-10-10T08:35:31Z" finished-at="2019-10-10T08:35:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="titletest()[pri:1, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="titletest" duration-ms="120" started-at="2019-10-10T08:35:37Z" finished-at="2019-10-10T08:35:37Z">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[Title does not match expected:<false> but was:<true>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: Title does not match expected:<false> but was:<true>
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.failNotEquals(Assert.java:282)
	at junit.framework.Assert.assertEquals(Assert.java:64)
	at junit.framework.Assert.assertEquals(Assert.java:149)
	at com.amazon.qa.testcases.LoginPageTest.titletest(LoginPageTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:969)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1148)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titletest -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="close" is-config="true" duration-ms="744" started-at="2019-10-10T08:35:37Z" finished-at="2019-10-10T08:35:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="setup" is-config="true" duration-ms="7216" started-at="2019-10-10T08:35:38Z" finished-at="2019-10-10T08:35:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="titletest()[pri:1, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="titletest" duration-ms="71" started-at="2019-10-10T08:35:45Z" finished-at="2019-10-10T08:35:45Z">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[Title does not match expected:<false> but was:<true>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: Title does not match expected:<false> but was:<true>
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.failNotEquals(Assert.java:282)
	at junit.framework.Assert.assertEquals(Assert.java:64)
	at junit.framework.Assert.assertEquals(Assert.java:149)
	at com.amazon.qa.testcases.LoginPageTest.titletest(LoginPageTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:969)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1148)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titletest -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="close" is-config="true" duration-ms="944" started-at="2019-10-10T08:35:45Z" finished-at="2019-10-10T08:35:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="setup" is-config="true" duration-ms="11259" started-at="2019-10-10T08:35:46Z" finished-at="2019-10-10T08:35:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="titletest()[pri:1, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="titletest" duration-ms="99" started-at="2019-10-10T08:35:58Z" finished-at="2019-10-10T08:35:58Z">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[Title does not match expected:<false> but was:<true>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: Title does not match expected:<false> but was:<true>
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.failNotEquals(Assert.java:282)
	at junit.framework.Assert.assertEquals(Assert.java:64)
	at junit.framework.Assert.assertEquals(Assert.java:149)
	at com.amazon.qa.testcases.LoginPageTest.titletest(LoginPageTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:969)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1148)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titletest -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="close" is-config="true" duration-ms="834" started-at="2019-10-10T08:35:58Z" finished-at="2019-10-10T08:35:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="setup" is-config="true" duration-ms="11335" started-at="2019-10-10T08:35:58Z" finished-at="2019-10-10T08:36:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="titletest()[pri:1, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="titletest" duration-ms="113" started-at="2019-10-10T08:36:10Z" finished-at="2019-10-10T08:36:10Z">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[Title does not match expected:<false> but was:<true>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: Title does not match expected:<false> but was:<true>
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.failNotEquals(Assert.java:282)
	at junit.framework.Assert.assertEquals(Assert.java:64)
	at junit.framework.Assert.assertEquals(Assert.java:149)
	at com.amazon.qa.testcases.LoginPageTest.titletest(LoginPageTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:969)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1148)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titletest -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="close" is-config="true" duration-ms="801" started-at="2019-10-10T08:36:10Z" finished-at="2019-10-10T08:36:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="FAIL" signature="setup()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="setup" is-config="true" duration-ms="0" started-at="2019-10-10T08:36:11Z" finished-at="2019-10-10T08:36:11Z">
          <exception class="org.openqa.selenium.SessionNotCreatedException">
            <message>
              <![CDATA[session not created
from chrome not reachable
  (Session info: chrome=77.0.3865.90)
Build info: version: '4.0.0-alpha-2', revision: 'f148142cf8', time: '2019-07-01T20:55:26'
System info: host: 'DESKTOP-CRT2G8C', ip: '192.168.0.112', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
	Ordinal0 [0x01547C93+1473683]
	Ordinal0 [0x014CADB1+961969]
	Ordinal0 [0x0145418C+475532]
	Ordinal0 [0x0144DB1A+449306]
	Ordinal0 [0x0144E1B8+451000]
	Ordinal0 [0x0144F2C5+455365]
	Ordinal0 [0x0144B4C5+439493]
	Ordinal0 [0x01454F40+479040]
	Ordinal0 [0x014098FE+170238]
	Ordinal0 [0x01408D1D+167197]
	Ordinal0 [0x0140742B+160811]
	Ordinal0 [0x013F0AE6+68326]
	Ordinal0 [0x013F1BF0+72688]
	Ordinal0 [0x013F1B89+72585]
	Ordinal0 [0x014E4B77+1067895]
	GetHandleVerifier [0x015E6875+506677]
	GetHandleVerifier [0x015E6610+506064]
	GetHandleVerifier [0x015ED1B8+533624]
	GetHandleVerifier [0x015E704A+508682]
	Ordinal0 [0x014DC266+1032806]
	Ordinal0 [0x014DC0DF+1032415]
	Ordinal0 [0x014E6A8B+1075851]
	Ordinal0 [0x014E6BF3+1076211]
	Ordinal0 [0x014E5BA5+1072037]
	BaseThreadInitThunk [0x762E0419+25]
	RtlGetAppContainerNamedObjectPath [0x7795662D+237]
	RtlGetAppContainerNamedObjectPath [0x779565FD+189]
]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.SessionNotCreatedException: session not created
from chrome not reachable
  (Session info: chrome=77.0.3865.90)
Build info: version: '4.0.0-alpha-2', revision: 'f148142cf8', time: '2019-07-01T20:55:26'
System info: host: 'DESKTOP-CRT2G8C', ip: '192.168.0.112', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: driver.version: ChromeDriver
remote stacktrace: Backtrace:
	Ordinal0 [0x01547C93+1473683]
	Ordinal0 [0x014CADB1+961969]
	Ordinal0 [0x0145418C+475532]
	Ordinal0 [0x0144DB1A+449306]
	Ordinal0 [0x0144E1B8+451000]
	Ordinal0 [0x0144F2C5+455365]
	Ordinal0 [0x0144B4C5+439493]
	Ordinal0 [0x01454F40+479040]
	Ordinal0 [0x014098FE+170238]
	Ordinal0 [0x01408D1D+167197]
	Ordinal0 [0x0140742B+160811]
	Ordinal0 [0x013F0AE6+68326]
	Ordinal0 [0x013F1BF0+72688]
	Ordinal0 [0x013F1B89+72585]
	Ordinal0 [0x014E4B77+1067895]
	GetHandleVerifier [0x015E6875+506677]
	GetHandleVerifier [0x015E6610+506064]
	GetHandleVerifier [0x015ED1B8+533624]
	GetHandleVerifier [0x015E704A+508682]
	Ordinal0 [0x014DC266+1032806]
	Ordinal0 [0x014DC0DF+1032415]
	Ordinal0 [0x014E6A8B+1075851]
	Ordinal0 [0x014E6BF3+1076211]
	Ordinal0 [0x014E5BA5+1072037]
	BaseThreadInitThunk [0x762E0419+25]
	RtlGetAppContainerNamedObjectPath [0x7795662D+237]
	RtlGetAppContainerNamedObjectPath [0x779565FD+189]

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$1(ProtocolHandshake.java:127)
	at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
	at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
	at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:129)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:139)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:576)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:210)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:130)
	at org.openqa.selenium.chromium.ChromiumDriver.<init>(ChromiumDriver.java:73)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:163)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:150)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:105)
	at com.amzon.qa.base.TestBase.initialization(TestBase.java:45)
	at com.amazon.qa.testcases.LoginPageTest.setup(LoginPageTest.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:589)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:969)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1148)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.SessionNotCreatedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="titletest()[pri:1, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="titletest" duration-ms="1" started-at="2019-10-10T08:36:21Z" finished-at="2019-10-10T08:36:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titletest -->
        <test-method status="SKIP" signature="close()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@1cd072a9]" name="close" is-config="true" duration-ms="0" started-at="2019-10-10T08:36:21Z" finished-at="2019-10-10T08:36:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.amazon.qa.testcases.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
